lib/systems/parse.nix

```diff
@@ -86,7 +86,7 @@ rec {
     i486     = { bits = 32; significantByte = littleEndian; family = "x86"; };
     i586     = { bits = 32; significantByte = littleEndian; family = "x86"; };
```

Comment:
```
arch = "i486";
arch = "i586";
```
?

Reply:
where the name is the same it is not needed.